src/controller/CHIPDeviceControllerFactory.cpp

```diff
@@ -150,6 +158,36 @@ CHIP_ERROR DeviceControllerFactory::InitSystemState(FactoryInitParams params)
     ReturnErrorOnFailure(Dnssd::Resolver::Instance().Init(stateParams.udpEndPointManager));
 #endif // CHIP_DEVICE_CONFIG_ENABLE_DNSSD
 
+    if (params.enableServerInteractions)
+    {
+        stateParams.caseServer = chip::Platform::New<CASEServer>();
+
+        ReturnErrorOnFailure(stateParams.caseServer->ListenForSessionEstablishment(
+            stateParams.exchangeMgr, stateParams.transportMgr, DeviceLayer::ConnectivityMgr().GetBleLayer(), stateParams.sessionMgr,
```

Comment:
We should not pass in the BLE layer here.  If we do, establishing a CASE session to us will close all our BLE connections.  That's perhaps ok for a non-commissioner, but for a commissioner that might have a BLE connection while also doing operational things on the network, this is not desirable.

Just pass null, please.